gradle/check-licenses.gradle

```diff
@@ -15,6 +15,7 @@
  * The lists of the license we accept.
  */
 ext.acceptedLicenses = [
+  'Apache v2',
```

Comment:
Put this in the aliases section instead?

Reply:
Missed that. Fixed!